### PR TITLE
Enable dependabot

### PR DESCRIPTION
**Pull request by slifty**
This PR enables dependabot for this repository.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
```
Comment (cecilia-donnelly) [inline .github/dependabot.yml:8]: I would expect us to include `npm` here as part of this PR - were you intending this to be the skeleton without starting updates? Or am I misreading the docs?

**Review by cecilia-donnelly — COMMENTED** [review]
Thanks! One question inline.

File: .github/dependabot.yml (hunk 1 above)
Comment (slifty) [reply .github/dependabot.yml]: Whoops! I thought I did that but I clearly did not :D

**Review by cecilia-donnelly — APPROVED** [review]
Thanks, @slifty !

**Pull request merged**